docs/docs/intro.md

```diff
@@ -1,34 +1,44 @@
 # Introduction
-
-OpenAssistant is a chat-based assistant that understands tasks, can interact
-with third-party systems, and retrieve information dynamically to do so.
-
-It can be extended and personalized easily and is developed as free, open-source
-software.
-
-## Our Vision
-
-We want OpenAssistant to be the single, unifying platform that all other systems
-use to interface with humans.
+> The FAQ page is available at [here](https://projects.laion.ai/Open-Assistant/docs/faq).
+
+Open Assistant (abbreviated as OA) is a chat-based and open-source assistant. The vision of the project is to make a large language model that can run on a single high-end consumer GPU.
+
+While Open Assistant is often compared to ChatGPT, Open Assistant should be more natural to interact with and capable at dealing with sensitive topics. When adapted from the conversational form, Open Assistant can interface with other third-party applications easily as well as retrieve information from databases and the Internet.
```

Comment:
Where is this text from? I feel like it might be overpromising a little, especially with wording like "can" perhaps implying these goals are already achieved rather than being ambitions